src/main/kotlin/org/rust/cargo/runconfig/buildtool/CargoBuildManager.kt

```diff
@@ -278,15 +277,23 @@ object CargoBuildManager {
     }
 
     private val cargoBuildPatch: CargoPatch = { commandLine ->
-        val additionalArguments = commandLine.additionalArguments.toMutableList()
-        additionalArguments.remove("-q")
-        additionalArguments.remove("--quiet")
-        addFormatJsonOption(additionalArguments, "--message-format")
+        val additionalArguments = mutableListOf<String>().apply {
+            addAll(commandLine.additionalArguments)
+            remove("-q")
+            remove("--quiet")
+            val format = if (SystemInfo.isUnix) "json-diagnostic-rendered-ansi" else "json"
```

Comment:
Let's add a link or an explanation about differences between formats and why they differ for unix and windows

Reply:
Looks like `json-diagnostic-rendered-ansi` now works on Windows.